yt/visualization/tests/test_commons.py

```diff
@@ -36,3 +41,17 @@ def test_custom_valid_ext(name, suffix, expected):
 
     assert result1 == expected
     assert result2 == expected
+
+
+def test_extent_swap():
+    input_extent = [1, 2, 3, 4]
+    expected = [3, 4, 1, 2]
+    assert _swap_axes_extents(input_extent) == expected
+    assert _swap_axes_extents(tuple(input_extent)) == tuple(expected)
+
+
+def test_swap_arg_pair_order():
+    assert _swap_arg_pair_order(1, 2) == (2, 1)
+    assert _swap_arg_pair_order(1, 2, 3, 4, 5, 6) == (2, 1, 4, 3, 6, 5)
+    assert_raises(AssertionError, _swap_arg_pair_order, 1)
+    assert_raises(AssertionError, _swap_arg_pair_order, 1, 2, 3)
```

Comment:
If you're actually _testing_ this then you just convinced me it _should_ be changed to a `TypeError` internally